io/include/pcl/io/grabber.h

```diff
@@ -69,7 +69,7 @@ namespace pcl
         * \return Connection object, that can be used to disconnect the callback method from the signal again.
         */
       template<typename T> boost::signals2::connection 
-      registerCallback (const boost::function<T>& callback);
+      registerCallback (const std::function<T>& callback);
```

Comment:
Wasn't the idea to add `FunctionT` template parameter when we change `boost` to `std` in public API? Same way you did in the very first commit of this PR?

Reply:
I forgot about it. I'll go over it tomorrow.